src/pybind/mgr/cephadm/migrations.py

```diff
@@ -12,7 +13,7 @@
 if TYPE_CHECKING:
     from .module import CephadmOrchestrator
 
-LAST_MIGRATION = 5
+LAST_MIGRATION = 6
```

Comment:
It looks like one of the teuthology tests has a hardcoded check for a migration current of 5 at the end of an upgrade. I guess we'll have to bump that check to 6 as part of this PR in order to have the test pass.
```
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/run_tasks.py", line 103, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/run_tasks.py", line 82, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_6f0f906272c8fb053c785bf0340c636123037ba1/qa/tasks/cephadm.py", line 1230, in shell
    _shell(ctx, cluster_name, remote,
  File "/home/teuthworker/src/git.ceph.com_ceph-c_6f0f906272c8fb053c785bf0340c636123037ba1/qa/tasks/cephadm.py", line 39, in _shell
    return remote.run(
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/orchestra/remote.py", line 525, in run
    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/orchestra/run.py", line 455, in run
    r.wait()
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/orchestra/run.py", line 161, in wait
    self._raise_for_status()
  File "/home/teuthworker/src/git.ceph.com_teuthology_5c9057f4b1ecae7b7410e418e40b739ff0616d25/teuthology/orchestra/run.py", line 181, in _raise_for_status
    raise CommandFailedError(
teuthology.exceptions.CommandFailedError: Command failed on smithi170 with status 1: "sudo /home/ubuntu/cephtest/cephadm --image quay.io/ceph/ceph:v16.2.0 shell -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring --fsid a52a8a5e-c8bc-11ed-9afe-001a4aab830c -- bash -c 'ceph config get mgr mgr/cephadm/migration_current | grep 5'"
```
The check seems to be in https://github.com/ceph/ceph/blob/main/qa/suites/orch/cephadm/mgr-nfs-upgrade/4-final.yaml

Reply:
https://pulpito.ceph.com/adking-2023-03-15_01:34:51-orch:cephadm-wip-adk-testing-2023-03-14-1905-distro-default-smithi/7208424/

Reply:
@adk3798 Thanks Adam. I increased the max migration to 6.